cylc/flow/scheduler.py

```diff
@@ -708,8 +708,58 @@ def info_get_graph_raw(self, cto, ctn, group_nodes=None,
             self.config.feet)
 
     def info_get_task_requisites(self, items, list_prereqs=False):
-        """Return prerequisites of a task."""
-        return self.pool.get_task_requisites(items, list_prereqs=list_prereqs)
+        """Return prerequisites and outputs etc. of a task.
+
+        Result in a dict of a dict:
+        {
+            "task_id": {
+                "meta": {key: value, ...},
+                "prerequisites": {key: value, ...},
+                "outputs": {key: value, ...},
+                "extras": {key: value, ...},
+            },
+            ...
+        }
+        """
+        itasks, bad_items = self.pool.filter_task_proxies(items)
+        results = {}
+        now = time()
+        for itask in itasks:
+            if list_prereqs:
+                results[itask.identity] = {
+                    'prerequisites': itask.state.prerequisites_dump(
+                        list_prereqs=True)}
+                continue
+            extras = {}
+            if itask.tdef.clocktrigger_offset is not None:
+                extras['Clock trigger time reached'] = (
+                    not itask.is_waiting_clock(now))
+                extras['Triggers at'] = get_time_string_from_unix_time(
```

Comment:
My IDE is complaining about `get_time_string_from_unix_time`. 

I guess we need to `from cylc.flow.wallclock import get_time_string_from_unix_time` or something similar?

The Travis builds failed in all functional tests, but the unit tests passed. I am not sure if the functional tests failures are related to this issue and maybe a crash while the loop is running... if not, then it probably means we need a unit test to cover this part too :grimacing:

Reply:
Fixed the import problem.

`scheduler.py` has almost no unit test coverage yet. I think I'll ignore that for now as off-topic on this branch (the code only appears here because I moved the method from one module to another).